Tutorials/start/configuring-faraday.md

```diff
@@ -1,83 +1,69 @@
- * `DEFAULT_ALTITUDE_UNITS`Altitude to default to when no GPS is present or is not used
- * `GPS_BOOT_BIT` ON/OFF to allow GPS to turn on at boot
- * `GPS_PRESENT_BIT` Boolean value to inform Faraday whether there is a GPS present or not
-
-`[TELEMETRY]`
- * `UART_TELEMETRY_BOOT_BIT` ON/OFF sending telemetry over UART after boot
- * `RF_TELEMETRY_BOOT_BIT` ON/OFF sending telemetry over RF after boot
- * `TELEMETRY_DEFAULT_UART_INTERVAL` UART telemetry interval (seconds)
- * `TELEMETRY_DEFAULT_RF_INTERVAL` RF telemetry beacon interval (seconds)
-
-## Configuring Hardware
-
-We are almost there! Eventually this will be automated but for now this is what we have. The following steps will start the Device Configuration application server and then send a POST command to it in order to initiate programming. Please ensure Proxy is running prior to these steps.
-
-1. Navigate to `deviceconfiguration` folder in Explorer or terminal
-2. Run `deviceconfiguration.py`
-  * Windows: double-click on `deviceconfiguration.py`
-  * Linux: `python deviceconfiguration.py`
-3. Run `simpleconfig.py` to send configuration data to Faraday
-4. Press `ctrl+c` to exit simpleconfig when complete after reviewing changes
-5. Close `deviceconfiguration.py` window
-
-Successful operation of `simpleconfig.py` will print out the Flash contents Faraday is programmed with. After successful programming the script queries Faraday over UART to send its Flash contents so we can confirm proper programming.
-
-![Simpleconfig.py output](images/simpleconfig.png)
-
-Note:
- * Some fields such as `BOOT_FREQUENCY_MHZ` and bitmasks return MSP430 specific values which differ from configuration values or bitmasks.
+Now we should configure Faraday itself. The device configuration data is stored in the CC430 Flash memory and is used to store the callsign, ID, static GPS location, telemetry interval, among other settings.
+
+The `faraday-deviceconfiguration` program communicates with the Proxy server enabling easy configuration of Faraday hardware. Let's use it to give your Faraday radio a callsign and other necessary configuration values. Additionally, we use `faraday-simpleconfig` to kick off the programming actions and display relevant information back to you.
+
+## Initializing Device Configuration
+Just as with Proxy, we need to initialize the configuration file for `faraday-deviceconfiguration`. To do this run the following command:
+* `faraday-deviceconfiguration --init-config`
+
+Additionally we need to initialize the faraday radio configuration file. This is nearly the same:
+* `faraday-deviceconfiguration --init-faraday-config`
+
+![Device Configuration Initialization](images/deviceconfiguration-init.jpg)
+
+## Configuring Device Configuration
+Now we need to tell the `faraday-deviceconfiguration` program where to find Proxy and what to configure the radio with. At this time the resulting command can be long so please be careful. Here are the configuration options:
```

Comment:
Created https://github.com/FaradayRF/Faraday-Software/issues/225 to address this ✔️